infra/main.bicep

```diff
@@ -48,6 +48,11 @@ param azureOpenAiApiVersion string = ''
 
 param openAiServiceName string = ''
 param openAiResourceGroupName string = ''
+
+param speechResourceGroupName string = ''
+param speechResourceGroupLocation string = location
```

Comment:
Recommend adding AZURE_SPEEECH_LOCATION so that existing speech services can be used